provider/barebones-go/project-generate.toml

```diff
@@ -0,0 +1,22 @@
+[template]
+raw = [
+  ".goreleaser.yml"
+]
+
+[[placeholders]]
+name = "vendor_name"
+type = "string"
+prompt = "Vendor name?"
+default = "Acme"
+
+[[placeholders]]
+name = "capability_id"
+type = "string"
+prompt = "Capability ID?"
+default = "wasmcloud:httpserver"
+
+[[placeholders]]
+name = "go_mod_prefix"
+type = "string"
+prompt = "Go Module name prefix (will add name to end)?"
```

Comment:
nit: maybe the ? would be better before the parenthesis? I read it wrong but not sure if that's just me